app/code/community/Metrilo/Analytics/Api/Client.php

```diff
@@ -19,7 +20,7 @@ public function customer($customer)
         $validCustomer = $this->_validator->validateCustomer($customer);
         
         if ($validCustomer) {
-            return $this->backendCall('/customer', ['params' => $customer]);
+            return $this->backendCall('/v2/customer', ['params' => $customer]);
```

Comment:
Add a `namespace` variable in here. You'll need to change the version of the API in the future. Make your life easier.

Reply:
Added endpoint path as private variables in client class